scss/main.scss

```diff
@@ -222,6 +222,34 @@ textarea {
   }
 }
 
+// New Pulse Styles
+
+.featured-flag {
+  text-transform: uppercase;
+  color: white;
+  letter-spacing: 0.7px;
+  font-size: 9px;
+  font-weight: 700;
+
+  &::before {
+    content: "";
+    margin-right: 3px;
+    float: left;
+    background: url("../assets/svg/featured.svg") no-repeat 0 0 / contain;
+    width: 12px;
+    height: 12px;
+  }
+
+  &-container {
+    background: $blue;
+    position: absolute;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    font-size: 9px;
```

Comment:
we can remove this rule since we already have the same rule set for `.featured-flag`?

Reply:
I'm confused :/ This is the rule for the `.featured-flag-container` (the purple box). What did you mean by 'the same rule set' and where?

Reply:
yep, you're right! thanks for pointing that out.